fury/ui.py

```diff
@@ -5240,3 +5240,165 @@ def _set_position(self, coords):
+            Absolute coordinates (x, y) of the lower-left corner of the
+            UI component
+        size : (int, int)
+            Width and height of the pixels of this UI component.
+        font_size: (int)
+            size of the font
+        text_color: (float , float , float)
+            color of the text
+        color: (float, float, float)
+            Background color of card
+        """
+
+        self.image_path = image_path
+        self.body = body
+        self.title = title
+        self.card_size = size
+        self.padding = padding
+        self.text_color = text_color
+        self.text_scale = np.clip(1 - image_scale, 0, 1)
+        self._image_size = (self.card_size[0], self.card_size[1] *
+                            np.clip(image_scale, 0, 1))
+
+        super(Card2D, self).__init__()
+        self.panel.position = position
+
+    def _setup(self):
+        """Setup this UI component
+        Create an image widget
+        Create a title and body TextBlock2D widget
+        Create a Panel2D widget
+        Add all widgets in Panel2D widget
+        """
+
+        (card_width, card_height) = self.card_size
+        (_, image_height) = self._image_size
+        _text_box_size = (card_width - 2 * self.padding, (card_height *
+                          self.text_scale / 2) - (2 * self.padding))
+
+        self.image = ImageContainer2D(img_path=self.image_path,
+                                      size=self._image_size)
+
+        self.body_box = TextBlock2D(text=self.body,
+                                    size=_text_box_size,
+                                    color=self.text_color)
+
+        self.title_box = TextBlock2D(text=self.title, bold=True,
+                                     size=_text_box_size,
+                                     color=self.text_color)
+
+        _img_coords = (0, int(card_height - image_height))
+        _rem_space = int(_img_coords[1] / 3)
+        _title_coords = (self.padding, _rem_space - self.padding)
+        _text_coords = (self.padding, int(_title_coords[1] - _rem_space) -
+                        self.padding)
+
+        self.panel = Panel2D(size=self.card_size, color=(0.5, 0.5, 0.5))
+        self.panel.add_element(self.image, _img_coords)
+        self.panel.add_element(self.title_box, _title_coords)
+        self.panel.add_element(self.body_box, _text_coords)
+
+    def _get_actors(self):
+        """ Get the actors composing this UI component.
+        """
+
+        return self.panel.actors + self.image.actors \
+            + self.title_box.actors + self.body_box.actors
+
+    def _add_to_scene(self, _scene):
+        """ Add all subcomponents or VTK props that compose this UI component.
+        Parameters
+        ----------
+        scene : scene
+        """
+
+        self.panel.add_to_scene(_scene)
+        self.image.add_to_scene(_scene)
+        self.title_box.add_to_scene(_scene)
+        self.body_box.add_to_scene(_scene)
+
```

Comment:
No need to add these components to the scene as you have already added them to the panel.